ClientRuntimes/CSharp/ClientRuntime.Log4Net/ClientRuntime.Log4Net.csproj

```diff
@@ -6,18 +6,23 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <LibraryFxTarget Condition="'$(LibraryFxTarget)' == ''">net45</LibraryFxTarget>
     <ProjectGuid>{2EFD6648-39B4-476A-B302-8BC236F5F1B8}</ProjectGuid>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Configuration Condition=" '$(Configuration)' == '' ">Net40-Debug</Configuration>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Rest.Tracing.Log4Net</RootNamespace>
-    <AssemblyName>Microsoft.Rest.ClientRuntime.Log4Net</AssemblyName>
     <OutputType>Library</OutputType>
     <RestorePackages>true</RestorePackages>
     <NuGetPackageImportStamp>e31a3ee2</NuGetPackageImportStamp>
+    <AssemblyName>Microsoft.Rest.ClientRuntime.Log4Net</AssemblyName>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)' == 'Net40-Debug'">
     <DocumentationFile>bin\Net40-Debug\Microsoft.Rest.ClientRuntime.Log4Net.xml</DocumentationFile>
   </PropertyGroup>
-  <Import Project="$(SolutionDir)\tools\AutoRest.Settings.targets" />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Net40-Debug|AnyCPU' ">
```

Comment:
remove this, and the other "net40-debug" section above. This is redundant as the $(SolutionDir)\Tools\AutoRest.Settings.targets should cover all. If you have to add it, log an issue i can take a look